src/main/java/org/amahi/anywhere/util/Downloader.java

```diff
@@ -66,6 +67,12 @@ private void setUpDownloadReceiver() {
     }
 
     private void startDownloading(Uri downloadUri, String downloadName) {
+        
+        ///code to delete the file if it already exists
+        File file = new File(Environment.getExternalStorageDirectory()+"/"+Environment.DIRECTORY_DOWNLOADS+"/"+downloadName);
+        if(file.exists());
```

Comment:
This is an empty if statement. That is it does not make any difference if the condition is true or false, the line below this will be called whatsoever.
Note the `;` you have put at the end of if statement.

Reply:
Also please format the code properly before committing. Check this answer [here](https://stackoverflow.com/a/16580200/6352482).